### PR TITLE
Prepend forward slash for path in generated OAS definition

### PR DESCRIPTION
**Pull request by hemikak**

### CODE REVIEW

File: misc/swagger-ballerina/modules/ballerina-to-swagger/src/test/java/org/ballerinalang/ballerina/swagger/convertor/service/SwaggerResourcesTest.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    /**
+     * Test case to check if path value in ResourceConfig annotation having no prefixed forward slash generates valid
+     * OAS definition.
+     */
+    @Test(description = "Test forward slash is added when forward slash is not in the resource path value")
+    public void testPathNoPrefixedForwardSlash() throws SwaggerConverterException, IOException {
+        String balSrc =
+                FileUtils.readFileToString(balFilesPath.resolve("path-annotations-no-forward-slash.bal").toFile());
+        String oasSrc = FileUtils.readFileToString(oasDefinitionsPath.resolve("path-annotation-no-forward-slash.yaml")
+                .toFile());
```
Comment (Kishanthan) [inline misc/swagger-ballerina/modules/ballerina-to-swagger/src/test/java/org/ballerinalang/ballerina/swagger/convertor/service/SwaggerResourcesTest.java]: Lets break new lines keeping the period (.) on the same line as a best practice.

File: misc/swagger-ballerina/modules/ballerina-to-swagger/src/test/java/org/ballerinalang/ballerina/swagger/convertor/service/SwaggerResourcesTest.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    /**
+     * Test case to check if path value in ResourceConfig annotation having a prefixed forward slash generates valid
+     * OAS definition.
+     */
+    @Test(description = "Test forward slash is added when forward slash is not in the resource path value")
+    public void testPathWithPrefixedForwardSlash() throws SwaggerConverterException, IOException {
+        String balSrc =
+                FileUtils.readFileToString(balFilesPath.resolve("path-annotations-with-forward-slash.bal").toFile());
+        String oasSrc = FileUtils.readFileToString(oasDefinitionsPath.resolve("path-annotation-no-forward-slash.yaml")
+                .toFile());
```
Comment (Kishanthan) [inline misc/swagger-ballerina/modules/ballerina-to-swagger/src/test/java/org/ballerinalang/ballerina/swagger/convertor/service/SwaggerResourcesTest.java]: Lets break new lines keeping the period (.) on the same line as a best practice.

Comment (hemikak) [reply misc/swagger-ballerina/modules/ballerina-to-swagger/src/test/java/org/ballerinalang/ballerina/swagger/convertor/service/SwaggerResourcesTest.java]: Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/10449/commits/404a6ce463b4b8dedd46e773a60f1d287f20b835

File: misc/swagger-ballerina/modules/ballerina-to-swagger/src/test/java/org/ballerinalang/ballerina/swagger/convertor/service/SwaggerResourcesTest.java (hunk 1 above)
Comment (hemikak) [reply misc/swagger-ballerina/modules/ballerina-to-swagger/src/test/java/org/ballerinalang/ballerina/swagger/convertor/service/SwaggerResourcesTest.java]: Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/10449/commits/404a6ce463b4b8dedd46e773a60f1d287f20b835

File: misc/swagger-ballerina/modules/ballerina-to-swagger/src/test/resources/bal-files/path-annotations-with-forward-slash.bal [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        methods: ["GET"],
+        path: "/sayHelloAA"
+    }
+    sayHelloAA(endpoint caller, http:Request req) {
+        http:Response res = new;
+        res.setPayload("Hello, World!");
+        caller->respond(res) but { error e => log:printError(
+                                                  "Error sending response", err = e) };
+    }
+}
```
[the remark this answers is not in the document]
Comment (hemikak) [reply misc/swagger-ballerina/modules/ballerina-to-swagger/src/test/resources/bal-files/path-annotations-with-forward-slash.bal]: Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/10449/commits/404a6ce463b4b8dedd46e773a60f1d287f20b835

File: misc/swagger-ballerina/modules/ballerina-to-swagger/src/test/resources/bal-files/path-annotations-no-forward-slash.bal [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        methods: ["GET"],
+        path: "sayHelloAA"
+    }
+    sayHelloAA(endpoint caller, http:Request req) {
+        http:Response res = new;
+        res.setPayload("Hello, World!");
+        caller->respond(res) but { error e => log:printError(
+                                                  "Error sending response", err = e) };
+    }
+}
```
[the remark this answers is not in the document]
Comment (hemikak) [reply misc/swagger-ballerina/modules/ballerina-to-swagger/src/test/resources/bal-files/path-annotations-no-forward-slash.bal]: Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/10449/commits/404a6ce463b4b8dedd46e773a60f1d287f20b835

**Pull request merged**